guides/doc-Release_Notes/topics/foreman-3.10.0.adoc

```diff
@@ -0,0 +1,130 @@
+= Foreman 3.10.0
```

Comment:
Need to add deprecation for `foreman-maintain backup snapshot`

Reply:
Do you track this or should I open an issue for foreman-documentation?